modules/API/API.cpp

```diff
@@ -579,9 +585,10 @@ void API::init() {
                     try {
                         seconds = std::stoi(data);
                     } catch (const std::exception& e) {
-                        EVLOG_error
-                            << "Could not parse connector duration value for uk_random_delay_set_max_duration_s: "
-                            << e.what();
+                        EVLOG_error << "Could not parse connector duration value for "
+                                       "uk_random_delay_set_max_duration_s, using default value of "
+                                    << seconds << " seconds "
+                                    << ", error: " << e.what();
```

Comment:
These could be collapsed, and extra space before comma removed.
```suggestion
                                    << seconds << " seconds, error: " << e.what();
```